### PR TITLE
Add Elasticsearch connector to registry

### PR DESCRIPTION
**Pull request by darshan-lukhi**

### CODE REVIEW

File: registry/elasticsearch/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+| Native Queries + Logical Models | ❌         |       |
+| Simple Object Query             | ✅         |       |
+| Filter / Search                 | ✅         |       |
+| Simple Aggregation              | ✅         |       |
+| Sort                            | ✅         |       |
+| Paginate                        | ✅         |       |
+| Nested Objects                  | ✅         |       |
+| Nested Arrays                   | ✅         |       |
+| Nested Filtering                | ✅         |       |
+| Nested Sorting                  | ✅         |       |
```
Comment (codedmart) [inline registry/elasticsearch/README.md:28]: We don't have this yet in the engine. Did you mean to check them green?

Comment (codedmart) [reply registry/elasticsearch/README.md:28]: Specifically nested sorting and filtering.

**Comment by codedmart** [comment]
Is it elasticsearch or elastic?

**Comment by gneeri** [comment]
> Is it elasticsearch or elastic?

Here's what the oracular ChatGPT has to say about it:

> Elasticsearch is the specific name of the search and analytics engine. "Elastic" refers to the company that created Elasticsearch and offers a suite of related products and services. In summary:

File: registry/elasticsearch/README.md (hunk 1 above)
Comment (gneeri) [reply registry/elasticsearch/README.md:28]: No, these should not be checked and are not checked in the README on the repo. I'll update that here. Thanks for catching that.

Comment (gneeri) [reply registry/elasticsearch/README.md]: This should now be sorted.

**Comment by codedmart** [comment]
OK fair just wanted to be sure.

**Review by codedmart — APPROVED** [review]

**Pull request merged**